ipython/find_parameter_sources_and_assign_uncertainties.ipynb

```diff
@@ -0,0 +1,361 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import copy\n",
+    "\n",
+    "import numpy\n",
+    "from IPython.display import display\n",
+    "\n",
+    "from rmgpy.tools.uncertainty import Uncertainty, ThermoParameterUncertainty, KineticParameterUncertainty"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Parameter Uncertainty Assignment\n",
+    "\n",
+    "This script aims to extract model sources in a clear and informative format.  The script first shows what all the kinetic and thermo sources are in a model. Then it goes through each reaction and species to show their source and what the assigned uncertainties are.  This can be used with any RMG-generated CHEMKIN file that is annotated."
+   ]
```

Comment:
Should we update this comment to include a warning that this will only work if the model was generated with a version of RMG where the uncertainty comments are compatible with this version of RMG?